onboard-jmuniz/app/(home)/users.tsx

```diff
@@ -0,0 +1,31 @@
+import { UsersListElementContainer } from "@/components/users-list/users-list-element-container";
+import useGetUsersList from "@/hooks/useGetUsersList";
+import React from "react";
+import { FlatList, Text, View } from "react-native";
+
+export interface UserProps {
+  name: string;
+  email: string;
+}
+
+export default function UsersView() {
+  const users = useGetUsersList(0);
+  const UserElement = (props: UserProps) => {
+    return (
+      <UsersListElementContainer>
+        <Text>Name: {props.name}</Text>
+        <Text>Email: {props.email}</Text>
+      </UsersListElementContainer>
+    );
+  };
+  return (
+    <View>
+      <FlatList
+        data={users}
+        renderItem={({ item }) => (
+          <UserElement name={item.name} email={item.email} />
+        )}
+      ></FlatList>
```

Comment:
```suggestion
      />
```

Quando não tem children dá pra fechar a tag direto.

Reply:
Beleza, isso foi alterado na próxima PR.